tests/integration/models/reddit/test_subreddit.py

```diff
@@ -229,14 +270,30 @@ def test_unignore_reports(self):
 
     def test_unread(self):
         self.reddit.read_only = False
-        with self.recorder.use_cassette(
-                'TestSubredditModeration.test_unread'):
+        with self.recorder.use_cassette('TestSubredditModeration.test_unread'):
             count = 0
             for item in self.reddit.subreddit('all').mod.unread():
                 assert isinstance(item, SubredditMessage)
                 count += 1
             assert count > 0
 
+    @mock.patch('time.sleep', return_value=None)
+    def test_update_settings(self, _):
+        self.reddit.read_only = False
+        with self.recorder.use_cassette(
+                'TestSubredditModeration.test_update_settings'):
+            before_settings = self.subreddit.mod.settings()
+            self.subreddit.mod.update_settings(title='A')
+            assert self.subreddit.title == 'A'
+            after_settings = self.subreddit.mod.settings()
+
+            # Ensure that nothing has changed besides what was specified.
+            del before_settings['title']
```

Comment:
No need to change it but you could do `before_settings['title']  = 'A'` to save one line :)